documentation/en/api-v2-unstable-methods.md

```diff
@@ -0,0 +1,56 @@
+# Groups
+* [Chain](#Chain)
+  * [ChainGetTipSet](#ChainGetTipSet)
+## Chain
```

Comment:
I didn't see a quick existing way to add a comment to something like `https://www.notion.so/filecoindev/Lotus-F3-aware-APIs-1cfdc41950c180ae97fef580e79427d5` in `docsgen.go` before or after the table of contents.  If it is easy, maybe we do so to tie togther docs work happening outside of the repo.  No worries if not.  Feel free to skip/resolve this comment.

Reply:
Added links to notion doc in generated markdown: https://github.com/filecoin-project/lotus/pull/13003/commits/338572287f3c73d4d3112e37f7e528d137cd9650